benchmarks/dali_vs_python/BM_jasper/README.md

```diff
@@ -0,0 +1,6 @@
+The torchaudio based processing of audio files comes from NVIDIA Deep Learning Examples project:
+https://github.com/NVIDIA/DeepLearningExamples/
+
+jasper_python/1/features.py: https://github.com/NVIDIA/DeepLearningExamples/blob/master/PyTorch/SpeechRecognition/Jasper/common/features.py
+
+The audio sample used for benchmarks comes from [LibriSpeech](https://ieeexplore.ieee.org/document/7178964) dataset.
```

Comment:
Is one sample enough to benchmark? Perhaps we could make the benchmark pull those audio samples from the web, instead of having them in the repo?

Reply:
Starting from question 2, I believe it's more convenient to have them in the repository. When pulling the samples from the web, we'd probably need to pull whole LibriSpeech, uncompress it and then pick some samples. This makes setting up the benchmark problematic. Our main idea in the DALI Backend is that all the banchmarks and tests should be as-easy-as-possible to run locally.

And as for the question 1: the `perf_analyzer` runs multiple processing requests through the server in order to stabilize the time readings. Every such requests consists of a batch of samples. AFAIK, DALI does not use any previous-processed heuristic, so even if a batch consists of a 256 copies of one sample, the processing is conducted as usual. So I believe, that adding more samples would impact only the amount of memory allocated by DALI and not necessarily impact the processing. This would make having more samples than one rather unnecessary. @banasraf , what's your opinion?

Reply:
My point is that we are using the same sample (same length, for example). Should we add some variability?